sources/packages/web/src/types/contracts/institution/InstitutionUser.ts

```diff
@@ -29,7 +29,7 @@ export interface InstitutionUserRoleLocation {
   locationId?: number;
   userType?: string;
 }
-export interface InstitutionUserDto {
+export interface InstitutionUserAPIOutDTO {
```

Comment:
Please add a blank line.

Reply:
I'm removing this interface as I didn't find any reference to this one. There's another DTO with the same name.